src/one_dragon/utils/log_utils.py

```
@@ -94,11 +59,67 @@ def get_or_create_logger(name: str, config: LoggerConfig | None = None) -> loggi
+    project_logger_name: str,
+    project_log_file_path: str,
+    framework_log_file_path: str,
+    *,
+    level: int = logging.INFO,
+    project_add_console_handler: bool = False,
+    framework_add_console_handler: bool = False,
+    framework_logger_name: str = LOGGER_NAME,
+) -> ProjectRuntimeLoggingContext:
+    """为项目运行态显式启用项目日志与框架日志分流。
+
+    默认的框架日志仍然写入 `log.txt`；只有项目主动调用本函数时，
```

Ruff (0.15.11)

[warning] Docstring contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF002)